templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -7,4 +7,4 @@ <h1>Delete manufacturer?</h1>
 
     <input type="submit" value="Yes" class="btn btn-danger">
   </form>
```

Comment:
You should add a 'Cancel' button in your form. This button should redirect the user to the previous page they were on. This is a code style issue according to the checklist.